roles/data_managers/tasks/data_managers.yml

```diff
@@ -0,0 +1,28 @@
+- name: Create/invoke script virtualenv
+  pip: name={{ item }} virtualenv={{ galaxy_tools_base_dir }}/venv virtualenv_command="{{ pip_virtualenv_command | default( 'virtualenv' ) }}"
+  with_items:
+    - pyyaml
+    - bioblend
+
+- name: Place tool management script
+  copy: src=roles/galaxyprojectdotorg.galaxy-tools/files/install_tool_shed_tools.py dest={{ galaxy_tools_base_dir }}/install_tool_shed_tools.py
+
+- name: Copy data manager task file
+  copy: src={{ item }} dest={{ galaxy_tools_base_dir }}/
+  with_items:
+    - "{{ data_manager_task_file_list }}"
+
+- name: Run data managers
+  command: "{{ galaxy_tools_base_dir }}/venv/bin/python install_tool_shed_tools.py -d {{ item|basename }} -a {{ default_admin_api_key }} -g {{ galaxy_hostname }}"
+  with_items:
+    - "{{ data_manager_task_file_list }}"
+  args:
+    chdir: "{{ galaxy_tools_base_dir }}"
+
+- name: Remove tool management script
+  file: dest={{ galaxy_tools_base_dir }}/install_tool_shed_tools.py state=absent
+
+- name: Remove data manager task file
+  file: src={{ item }} dest={{ galaxy_tools_base_dir }}/ state=absent
```

Comment:
should be
```
  file: dest={{ galaxy_tools_base_dir }}/{{ item|basename }} state=absent
```
otherwise, /tmp is deleted and replaying the playbook fails